src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionLikelihoods.java

```diff
@@ -124,6 +126,7 @@ protected static double tau(final double lambda0, final double kappa) {
      * @param state allele fraction state
      * @param segment index of segment containing this het site
      * @param count AllelicCount of alt and ref reads
+     * @param allelicPON allelic panel of normals containing total alt and ref counts observed across all normals at each site
```

Comment:
javadoc for this parameter is misleading because the method uses the PoN for the processing it does *after* getting alt and refs counts.  Counts aren't part of the API.